android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -80,8 +80,6 @@ public boolean onPreferenceChange(Preference preference, Object newValue)
           mPrefLanguages.setEnabled(false);
         if (mLangInfo != null)
           mLangInfo.setSummary(R.string.prefs_languages_information_off);
-        if (mLangInfoLink != null && isOnTtsScreen())
-          getPreferenceScreen().addPreference(mLangInfoLink);
```

Comment:
Behaviour of `mLangInfoLink` is little bit strange. It's already on TTS Settings screen so don't need to add it at a runtime.

Reply:
Does it make sense to cleanup mLangInfoLink related code now, to make it easier in the future? E.g. initLangInfoLink can be removed, all null checks can be removed too. Maybe storing mLangInfoLink as a variable would not be needed at all.

Reply:
Renaming would be great. Simplifying the code and removing unnecessary vars and functions would be also great )

Reply:
Replaced field `mLangInfoLink` with local variable.
Renamed fields, method and local variable by adding `Tts`.